app/src/main/java/com/realityexpander/tasky/core/presentation/notifications/RemindAtNotificationManagerImpl.kt

```diff
@@ -36,33 +36,38 @@ import com.realityexpander.tasky.core.util.toZonedDateTime
 import logcat.logcat
 import java.time.format.DateTimeFormatter
 import java.util.*
+import javax.inject.Inject
+
+
+class RemindAtNotificationManagerImpl @Inject constructor(
```

Comment:
You dont need to add @Inject constructor

Since its being created on AppModule